extensions/amp-accordion/amp-accordion.md

```diff
@@ -45,38 +47,38 @@ Provides a way for viewers to glance at the content outline and jump to any sect
-- The second child (of the section) can be any tag allowed in AMP HTML and represents the content of the section.
-- Clicking/tapping on the heading of a section expands or collapses the section.
-- The collapsed/expanded state of each section in the `amp-accordion` element will be preserved for the session level. To opt out of preserving this state, add the `disable-session-states` attribute to the `amp-accordion` element.
+  - The first child in a `<section>` is the heading for that section of the `amp-accordion`. It must be a heading element: [`<h1>-<h6>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/heading_elements) or [`<header>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header).
+  - The second child in a `<section>` is the expandable/collapsable content. It can be any tag allowed in [AMP HTML](../../spec/amp-html-format.md).
+- Clicking/tapping on a `<section>` heading expands or collapses the section.
+- An `amp-accordion` with a defined `id` preserves the collapsed/expanded state of each section while the user remains on your domain.
 
 [/filter] <!-- formats="websites, ads" -->
 [filter formats="email"]
 
-- An `amp-accordion` can contain one or more `<section>` elements as its direct children.
+- An `amp-accordion` accepts one or more `<section>` elements as its direct children.
 - Each `<section>` must contain exactly two direct children.
-- The first child (of the section) represents the heading for the section and must be a heading element (one of `h1`, `h2`, ..., `h6`, `header`).
-- The second child (of the section) can be any tag allowed in AMP HTML and represents the content of the section.
-- Clicking/tapping on the heading of a section expands or collapses the section.
+  - The first child in a `<section>` is the heading for that section of the `amp-accordion`. It must be a heading element: [`<h1>-<h6>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/heading_elements) or [`<header>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header).
+  - The second child in a `<section>` is the expandable/collapsable content. It can be any tag allowed in [AMP for Email](../../spec/email/amp-email-html.md).
+- Clicking/tapping on a `<section>` heading expands or collapses the section.
 
 [/filter] <!-- formats="email" -->
 
-#### Example: Displaying an accordion
+#### Example
 
-In this example, we display three sections, where the third section is expanded on page load. Also, we opted out of preserving the collapsed/expanded state by setting `disable-session-states`.
+The example below contains an `amp-accordion` with three sections. Use of the `expanded` attribute on the third section expands it on page load. [filter formats="websites, ads"]Preserving the collapsed/expanded state is disable by including the `disable-session-state` attribute.[/filter] <!-- formats="websites, ads" -->
```

Comment:
s/state is disable/state is disabled

But maybe better (and not passive): "Including the `disable-session-state` attribute preserves the collapsed/expanded state.

Reply:
Also: "The `expanded` attribute on the third section expands it on page load."